content/tracks/side-tracks/physics-libraries/index.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "Physics Libraries",
+  "description": "Tutorials on the Matter.js, Toxiclibs, and Box2d physics libraries",
+  "videos": [
```

Comment:
@kfahn22 should we include challenges related to matter, toxiclibs and box2d here?